android/app/src/main/java/app/organicmaps/bookmarks/Holders.java

```diff
@@ -183,27 +183,23 @@ protected void setSize()
       mSize.setText(getSizeString());
     }
 
-    private String getSizeString()
-    {
+    private String getSizeString() {
       final Resources resources = mSize.getResources();
       final int bookmarksCount = mEntity.getBookmarksCount();
       final int tracksCount = mEntity.getTracksCount();
 
-      if (mEntity.size() == 0)
-        return getQuantified(resources, R.plurals.objects, 0);
+      final String bookmarks = getQuantified(resources, R.plurals.places, bookmarksCount);
```

Comment:
It would be great to avoid creating a string if it's not needed, by moving this code into the condition.